internal/torcx/paths.go

```diff
@@ -87,5 +87,10 @@ func (cc *CommonConfig) NextProfile() string {
 
 // ArchiveFilename is the filename (no directory) for the archive of an image.
 func (im *Image) ArchiveFilename() string {
```

Comment:
Are these functions even used (I don't think so)? If so, can they just use the `format` property of the Image struct?